web-admin/src/features/projects/status/ProjectParseErrors.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+  import Spinner from "@rilldata/web-common/features/entity-management/Spinner.svelte";
+  import {
+    ResourceKind,
+    SingletonProjectParserName,
+  } from "@rilldata/web-common/features/entity-management/resource-selectors";
+  import { EntityStatus } from "@rilldata/web-common/features/entity-management/types";
+  import { createRuntimeServiceGetResource } from "@rilldata/web-common/runtime-client";
+  import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
+
+  $: parserErrors = createRuntimeServiceGetResource(
```

Comment:
The intent here is to only show parse errors and not reconcile errors right? Because in my project this shows no errors but there are a couple resources with reconcile errors.

Reply:
Correct – those (resource-specific) reconcile errors should be shown in the table above in the "Error" column. Do you see them there?